src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.querySelector('#largeImg');
+const listImg = document.querySelector('#thumbs');
+
+listImg.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target;
+
+  if (!target) {
+    return;
+  }
+
+  if (target.tagName === 'IMG') {
+    mainImg.src = target.parentElement.href;
+  } else if (target.tagName === 'A') {
+    mainImg.src = target.href;
+  }
```

Comment:
The logic assumes that the `target` will always be an `IMG` or an `A` tag. If there are other elements within `#thumbs`, this logic might not handle them correctly. Consider adding a more robust check or handling for other possible elements.